public/locales/en/translation.json

```
@@ -1 +1,5 @@
-{}
+{
+  "aboutUs": "About us",
+  "aboutUsText": "It is a long established fact that a reader will be distracted bythe readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using  content hereh. It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of etters, as opposed to using Content h ereh.",
+  "readMoreButton": "Read More"
+}
```

can you make this one more specific ? like this ? 
don't change it now just answer me 
```suggestion
{
aboutPage:{
  "aboutUs": "About us",
  "aboutUsText": "It is a long established fact that a reader will be distracted bythe readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using  content hereh. It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of etters, as opposed to using Content h ereh.",
  "readMoreButton": "Read More"
}
```